guides/common/modules/proc_creating-network-or-image-based-hosts.adoc

```diff
@@ -0,0 +1,89 @@
+[id="creating-network-or-image-based-hosts_{context}"]
+= Creating Hosts on {CRname}
```

Comment:
Question: Shouldn't this be "in {CRname}"?

Reply:
Good question. The original name of this file was "Creating Hosts on a KVM Server", which is appropriate because KVM is a single RHEL server that you create VMs on (whereas RHV is a whole infrastructure setup so that section's current title of "on a Red Hat Virtualization Server" is quite incorrect). "in KVM" (without "server") works fine though, I think. 

For some reason all of the other compute resource chapters seem to follow a convention of:

Chapter name: Provisioning VMs/Cloud Instances *in* Product Name

Section name: Creating Hosts *on* Product Name

That seems weird and I'll aim to standardise it all to "in". 

Side note: I'd considered "Creating Product Name Hosts", but dropped it because we should 100% avoid that phrasing for RHV. In RHV, "hosts" refers specifically to hypervisors and not to VMs, so saying "RHV hosts" would seem like you're provisioning a physical hypervisor (the product name for one of RHV's hypervisor types is even Red Hat Virtualization Host).